src/site/stages/build/process-cms-exports/schemas/transformed/node-press_release.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-press_release'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['press_release'] },
+    title: { type: 'string' },
+    entityMetatags: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          __typename: { type: 'string' },
+          key: { type: 'string' },
+          value: { type: 'string' },
+        },
+        required: ['__typename', 'key', 'value'],
+      },
+    },
+    entityUrl: { $ref: 'EntityUrl' },
+    fieldAddress: {
+      type: 'object',
+      properties: {
+        locality: { type: 'string' },
+        administrativeArea: { type: 'string' },
+      },
+      required: ['locality', 'administrativeArea'],
+    },
+    fieldIntroText: { type: 'string' },
+    fieldOffice: { $ref: 'transformed/node-health_care_region_page' },
+    fieldPdfVersion: { $ref: 'Media' },
+    fieldPressReleaseContact: {
+      type: 'array',
+      maxItems: 1,
```

Comment:
This is awfully similar to `node-person_profile`.

Reply:
```bash
jq --slurp 'map(select(.type[0].target_id == "press_release")) | map(.field_press_release_contact) | map(select(. != []))' node.*.json | sed -nE 's/.*"target_uuid": "(.*)".*/\1/p' | xargs -I UUID jq '.type[0].target_id' node.UUID.json | sort | uniq
```
:point_up: tells me they all _are_, in fact, `node-person_profile`s.

Reply:
Right - in the raw they are all `node-person_profile`s, but the transformed versions differ slightly (the title) but this can be changed.

Reply:
It makes the test data a pain in the butt, but I think in the end, it's better if we keep all instances of child entities unmodified after they go through the transformer. That way one `node-person_profile` doesn't look different from another `node-person_profile` depending on where it's found. This will hopefully avoid confusion down the road.

As such, I think it makes sense to add `title` to the `node-person_profile` transformer.